### PR TITLE
Add a title to the homepage link

### PR DESCRIPTION
**Pull request by anurbol**
Added a title to the homepage link. 

## References
  Related to https://github.com/github/accessibility-audits/issues/2739

### CODE REVIEW

**Comment by anurbol** [comment]
I assessed a couple of other options as well:

- Adding `<title>` to the svg doesn't feel correct as the [issue](https://github.com/github/accessibility-audits/issues/2739) says we need to add title to the link and not to the image. 
- Adding `alt` attribute is inappropriate as it is intended for using on `img`

**Review by 20shivangi — COMMENTED** [review]
Can you please attach a vide of voiceover on Safari with this change once ?

**Comment by prabum3** [comment]
We already have a link to `npmjs.com` on the right side. I think it is fine to have the logo redirect to docs homepage and have the title `npm docs`.

File: theme/src/components/header.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ function Header({location, isSearchEnabled = true}) {
         bg="#333333"
       >
         <Flex alignItems="center">
-          <Link as={GatsbyLink} to="/" style={logoStyle} mr={3}>
+          <Link as={GatsbyLink} to="/" style={logoStyle} mr={3} title='NPM home'>
```
Comment (20shivangi) [inline theme/src/components/header.js:48]: Please change the title to `npm Docs`

Also, npm should always be lowercase and not NPM :  https://github.com/npm/cli#faq-on-branding

**Comment by amit-avit** [comment]
+1 to the comments to just make it as `npm Docs`, also do check once that the tool passes with updating the `title` else we might have to update `aria-label` instead.

**Comment by amit-avit** [comment]
@anurbol @prabum3 @20shivangi  It seems we have 2 `a` pointing to the docs homepage one for logo another for text `npm Docs`. I would suggest that we only have one `a` and move the logo inside the `a` and then the text would act as title and the logo would be aria-hidden.

**Comment by 20shivangi** [comment]
> @anurbol @prabum3 @20shivangi It seems we have 2 `a` pointing to the docs homepage one for logo another for text `npm Docs`. I would suggest that we only have one `a` and move the logo inside the `a` and then the text would act as title and the logo would be aria-hidden.

Yes @amit-avit , that's what I was saying that there will be duplication :D
We can go with this approach

**Comment by anurbol** [comment]
@amit-avit @prabum3 @20shivangi 

I've merge the two links into one (there is now single `a` instead of two). Because previously there were two independent elements - logo and title, they were positioned on an arbitrary height relative to each other, and now they are strictly vertically aligned in their common link element (which lead to the logo now being positioned a bit lower). Which I believe is a good thing. But if I am mistaken please let me know.

https://user-images.githubusercontent.com/3603448/196427424-9af4d6a0-769c-4394-a944-996cf2bbf7df.mp4

**Review by amit-avit — APPROVED** [review]
LGTM
@anurbol personally I feel the alignment is better now with your changes.

```
     _ 
    /|\      |   |
   /_|_\     | __|__
 ____|____   |   |
 \_o_o_o_/   |   \__/

```

File: theme/src/components/header.js (hunk 1 above)
Comment (anurbol) [reply theme/src/components/header.js]: Please see https://github.com/npm/documentation/pull/253#issuecomment-1282300049

**Review by 20shivangi — APPROVED** [review]

**Review by Seryozha95 — APPROVED** [review]

**Review by Seryozha95 — APPROVED** [review]

**Pull request merged**